ban/http/api.py

```diff
@@ -350,7 +350,7 @@ def get_version(self, identifier, ref):
         instance = self.get_object(identifier)
         version = instance.load_version(ref)
         if not version:
-            abort(404)
+            abort(404, error='Version reference `{}` not found'.format(ref))
```

Comment:
No strong point.
Just wanted to avoid `"Excaped \"double quotes\" in json strings"`, and found back ticks as the "neutral" option out of the endless simple or double quotes in python. :)
You'd prefer simple quotes?